src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.tsx

```diff
@@ -35,6 +45,20 @@ function CurrentOrgPlan() {
       <InfoMessageStripeCallback />
       {accountDetails?.plan ? (
         <div className="flex flex-col gap-4 sm:mr-4 sm:flex-initial md:w-2/3 lg:w-3/4">
+          {!!planUpdatedNotification?.variant && (
+            <Alert variant={planUpdatedNotification.variant}>
+              {scheduleStart && scheduledPhase?.quantity && (
+                <Alert.Title>Plan successfully updated.</Alert.Title>
+              )}
+              {!!planUpdatedNotification?.variant && (
```

Comment:
The check above on line 48 looks to be the same as this, are we able to remove this extra check here?